dev-libs/libdispatch/files/libdispatch-5.3.3-musl.patch

```diff
@@ -0,0 +1,39 @@
+--- swift-corelibs-libdispatch-swift-5.3.3-RELEASE.orig/dispatch/source.h
```

Comment:
1. Please use 'git-like patches', i.e. a/ and b/ instead of looong.orig/ and looong/
2. Include a link to the relevant Gentoo bug(s) at the top of the patch
3. Include a link to where it's been forwarded/reported upstream if applicable (please do)